client/src/components/ProductsList.js

```diff
@@ -2,23 +2,29 @@ import React, { Component } from "react";
 import axios from "axios";
 import { Link } from "react-router-dom";
 import SearchFilter from "./SearchFilter";
+import useStyles from "./ProductListStyles";
+import {
+  Typography,
+  TextField,
+  Grid,
+  Box,
```

Comment:
Are we going to use this on thee ProductListing page? If not, let's remove it later. :)

```
  Line 8:   'TextField' is defined but never used  no-unused-vars
  Line 9:   'Grid' is defined but never used       no-unused-vars
  Line 10:  'Box' is defined but never used        no-unused-vars
```

Reply:
i'll remove it to keep it clean:)

Reply:
👍